src/OSPSuite.UI/Services/DiagramZoomRectangleService.cs

```diff
@@ -116,9 +115,12 @@ private bool mouseIsOverChart(MouseEventArgs e)
 
       private DiagramCoordinates pointLocationAt(MouseEventArgs e)
       {
+         if (_chartControl.XYDiagram == null)
+            return null;
+
          try
          {
-            return _diagram.PointToDiagram(new Point(e.X, e.Y));
+            return _chartControl.XYDiagram.PointToDiagram(new Point(e.X, e.Y));
          }
          catch (Exception)
```

Comment:
I think I understood why we were getting that exception all the time. The refernce to _diagram was somehow obsolete when no curves where available. Very confusing but now I do not store any reference to the diagram. Instead always use the one from the chart